src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -1212,19 +1219,24 @@ class DeviceArgument(cli.Argument):
     )
     DEVICE_NAME = cli.ArgumentProperties(
         key="device_name",
-        flags=("--name",),
+        flags=("--name", "-n"),
         description="Name of the Device",
         argparse_kwargs={"required": True},
     )
     DEVICE_NAME_OPTIONAL = cli.ArgumentProperties.duplicate(
         DEVICE_NAME,
         argparse_kwargs={"required": False},
     )
-    DEVICE_UDID = cli.ArgumentProperties(
-        key="device_udid",
-        flags=("--udid",),
-        description="Device ID (UDID)",
-        argparse_kwargs={"required": True},
+    DEVICE_UDIDS = cli.ArgumentProperties(
+        key="device_udids_argument",
```

Comment:
Nowhere else we have `argument` suffix in the key name. Also this makes the Python API of actions rather ugly, especially when used with kwargs.